src/config/graphical-session/gnome.md

```diff
@@ -0,0 +1,34 @@
+# GNOME
+
+## Pre-installation
+
+GNOME requires a display server such as [Xorg](./xorg.md) or a
+[Wayland](./wayland.md) compositor.
+
+Install the `elogind` and `adwaita-icon-theme` packages, ensure the `dbus`
+service is enabled, and reboot for the changes to take effect.
+
+## Installation
+
+Install the `gnome` package for a GNOME environment which includes GNOME
+applications.
+
+A minimal GNOME environment can be created by installing the `gnome-session` and
+`gdm` packages. Note, however, that such an minimal environment may not work
+well.
+
+Starting `gnome-session` via `~/.xinitrc` provides an Xorg session.
+
+If you require ZeroConf support, install the `avahi` package and enable the
+`avahi-daemon` service.
```

Comment:
Should we link http://www.zeroconf.org/ ? Or is that a case of "you know it if you need it"?